learningresources/models.py

```diff
@@ -61,6 +61,7 @@ class StaticAsset(BaseModel):
     Holds static assets for a course (css, html, javascript, images, etc)
     """
     course = models.ForeignKey(Course)
+    path = models.CharField(max_length=256)  # path in static folder
```

Comment:
We should probably increase the max_length to 64k since Linux allows up to 4k for the most part and 64k would use all of 2 bytes